# tr-js-day30
Question: How do you handle and propagate errors in a Promise chain?
Answer: In a Promise chain, you can use the catch() method to handle errors at any point. If you don't handle the error within a catch() block, it will propagate to the next catch() in the chain.

to run use command : node app.js

